### PR TITLE
VNext

### PR DESCRIPTION
**Pull request by SebastianStehle**

### CODE REVIEW

File: .nuget/NuGet.Config [hunk 1]
```diff
@@ -1,11 +1,6 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
-
 <configuration>
-  <solution>
-    <add key="disableSourceControlIntegration" value="true" />
-  </solution>
   <packageSources>
-    <add key="nuget.org" value="https://api.nuget.org/v3/index.json" />
-    <add key="OrleansPrerelease" value="https://dotnet.myget.org/F/orleans-prerelease/api/v3/index.json" />
+    <add key="nuget.org" value="https://api.nuget.org/v3/index.json" protocolVersion="3" />
```
Comment (galvesribeiro) [inline .nuget/NuGet.Config]: You can drop this file altogether.

**Comment by galvesribeiro** [comment]
I think the only thing missing is the package naming

**Comment by SebastianStehle** [comment]
We would need a fourth package for our shared code. But right my priority for this minor improvements is very low, I would also publish a RTM without the restructuring.

**Comment by galvesribeiro** [comment]
The shared code don't need a package. You can do like we do in Orleans and link the common files across multiple projects.

But anyway, It was just my thoughts. If you feel you want to keep this as is, its up to you. 

I'm just suggesting the rename on all projects that I've touch so they can be aligned with 2.0 release and package structure in Orleans.

**Comment by SebastianStehle** [comment]
Sure, that would be another option. Don't understand me wrong, I am thankful for the feedback, but right now I would be very happy if I can just change some version strings and enter "dotnet pack -c Release" to powershell ;) .... but I am not the only maintainer.

**Comment by galvesribeiro** [comment]
Sure :)

**Pull request merged**